src/main/java/com/stevedutch/assignment14/web/ChannelController.java

```diff
@@ -9,10 +9,8 @@ public class ChannelController {
 
 	@GetMapping("/channel")
 	public String showChannel(ModelMap model) {
-		model.put("user", model);
```

Comment:
no need to expose whole model to a variable. The idea of putting values in the model via the ModelMap is for the frontend to be able to access it when the UI is rendered. Since you are not really using those values in your frontend, there's no need to expose anything